src/assets/i18n/zh.json

```diff
@@ -319,7 +319,9 @@
     "convertObjectiveValues": "转换目标值",
     "convertObjectiveValuesTooltip": "如果勾选，当类型改变时，目标值将被转换以保留物品数量",
     "locations": "地点",
-    "locationsTooltip": "为配方和设备选择允许使用的地点"
+    "locationsTooltip": "为配方和设备选择允许使用的地点",
+     "recipeProductivity": "配方产能加成",
```

Comment:
I think prettier is complaining about the extra space on line 323

Reply:
fixed